### PR TITLE
Cleanup repo to reflect current state of logging

### PR DESCRIPTION
**Pull request by jcantrill**
This PR cleans up the master branch to reflect the current state of development beyond 4.6 release

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @openshift/sig-logging

File: docs/deploy-event-router.md [hunk 1]
```diff
@@ -1,106 +0,0 @@
-# Deploying Event Router
```
Comment (ewolinetz) [inline docs/deploy-event-router.md:1]: isn't this still relevant for now?

File: docs/checking-efk-health.md [hunk 2]
```diff
@@ -1,123 +0,0 @@
-# Checking EFK Health
```
Comment (ewolinetz) [inline docs/checking-efk-health.md:1]: i feel like this is still relevant as well

File: docs/deploy-event-router.md (hunk 1 above)
Comment (jcantrill) [reply docs/deploy-event-router.md:1]: doc is being moved to CLO

File: docs/checking-efk-health.md (hunk 2 above)
Comment (jcantrill) [reply docs/checking-efk-health.md:1]: I though i reviewed this content and the checks didnt really apply and/or out of date.  Should likely be move to the various "troubleshooting" sections in EO or CLO

**Pull request merged**